libs/platform/src/lib/components/form/radio-group/radio-group.component.ts

```diff
@@ -1,30 +1,43 @@
 import {
-    EventEmitter,
+    AfterContentChecked,
+    AfterViewInit,
+    ChangeDetectorRef,
+    ChangeDetectionStrategy,
     Component,
     ContentChildren,
-    QueryList,
+    EventEmitter,
     Input,
-    ChangeDetectorRef,
-    ChangeDetectionStrategy,
-    AfterViewInit,
+    OnDestroy,
+    Optional,
     Output,
+    QueryList,
     Self,
-    Optional,
-    ViewChildren, AfterContentChecked, OnDestroy
+    ViewEncapsulation,
+    ViewChildren
 } from '@angular/core';
 import { NgControl, NgForm } from '@angular/forms';
 import { takeUntil } from 'rxjs/operators';
 import { Subject } from 'rxjs';
 import { RadioButtonComponent } from './radio/radio.component';
 import { CollectionBaseInput } from '../collection-base.input';
+import { FormFieldControl } from '../form-control';
+
+/**
+ * Radio group implementation based on the
+ * https://github.com/SAP/fundamental-ngx/wiki/Platform:-RadioGroup-Technical-Design
+ * documents.
+ *
+ */
 
 // Increasing integer for generating unique ids for radio components.
 let nextUniqueId = 0;
```

Comment:
missing type

Reply:
New Lint checks has rule to not to put type, if it is initialised.